src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizePagesFormatter.java

```diff
@@ -63,19 +65,9 @@ public String format(String value) {
         // Remove pages prefix
         String cleanValue = value.replace("pp.", "").replace("p.", "");
         // remove unwanted literals including en dash, em dash, and whitespace
-        cleanValue = cleanValue.replaceAll("\u2013|\u2014", "-").replaceAll(REJECT_LITERALS, "");
-        // try to find pages pattern
-        Matcher matcher = PAGES_DETECT_PATTERN.matcher(cleanValue);
-        if (matcher.matches()) {
-            // replace
-            if (Strings.isNullOrEmpty(matcher.group("endpage"))) {
-                return matcher.replaceFirst(SINGLE_PAGE_REPLACE_PATTERN);
-            } else {
-                return matcher.replaceFirst(PAGES_REPLACE_PATTERN);
-            }
-        }
-        // no replacement
-        return value;
+        value = EM_EN_DASH_PATTERN.matcher(cleanValue).replaceAll("-")
+                                  .replaceAll("[ ]*[-]+[ ]*", "--");
+        return unprotectTermsFormatter.format(value.trim());
```

Comment:
This now formats all kinds of text, e.g. `pages  1-50`, `some-text-with-dashes` and `{A}`. I think this is not desired. I would keep it conservative: it's probably more annoying for users if hand-formatted page numbers are destroyed than if some cases need to be formatted by the user itself.

The biblatex documentation is not specific about the the type of characters that are allowed, and I'm also not sure if `I-X` is correct or `I--X` for roman page numbers.

Reply:
Result of test-driven-development 😇. I will think more about it now.

Reply:
> I'm also not sure if `I-X` is correct or `I--X` for roman page numbers.

This reads more as a page range, too. Roman 9 is written IX (and not I-X), isn't it?